Library/Formula/norm.rb

```diff
@@ -0,0 +1,30 @@
+class Norm < Formula
+  desc "NACK-Oriented Reliable Multicast"
+  homepage "http://www.nrl.navy.mil/itd/ncs/products/norm"
+  url "http://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
```

Comment:
Please use the https version of the `homepage`:
`homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"`
(it's mixed content; something to fix upstream)

And the same for the `url`:
`url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"`